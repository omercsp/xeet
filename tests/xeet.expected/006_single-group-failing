Starting run
Running groups: failing

Running 002_fail                                             .......[Failed ] rc=1, allowed=2
Full output file: XEET_DIR/sanity/xeet.out/002_fail

Running 010_timeout                                          .......[Failed ]
Command '['sleep', '5']' timed out after 1 seconds
Full output file: XEET_DIR/sanity/xeet.out/010_timeout

Running 011_bad_output                                       .......[Failed ] Output mismatch
Diff head:
>>>>>
--- XEET_DIR/sanity/xeet.expected/011_bad_output
+++ XEET_DIR/sanity/xeet.out/011_bad_output
@@ -1 +1 @@
-good output
+bad output
<<<<<
Full diff file: XEET_DIR/sanity/xeet.out/011_bad_output.diff
Full output file: XEET_DIR/sanity/xeet.out/011_bad_output

Running 015c_post_command_fail                               .......[Failed ] post run failed
Post run command returned: 1
Post run stderr:

Full output file: XEET_DIR/sanity/xeet.out/015c_post_command_fail


Finished iteration (#0/0)
Failed: 002_fail, 010_timeout, 011_bad_output, 015c_post_command_fail

