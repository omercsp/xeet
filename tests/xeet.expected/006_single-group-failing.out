Starting run
Included groups: failing
Running tests: 002_fail, 010_timeout, 011_bad_output, 015c_post_command_fail

002_fail                                                     ....... [Failed ] rc=1, allowed=0
Test stdout/stderr: (Empty stdout), (Empty stderr)

010_timeout                                                  ....... [Failed ]
Command '['sleep', '5']' timed out after 1 seconds
Test stdout/stderr: (Empty stdout), (Empty stderr)

011_bad_output                                               ....... [Failed ] stdout mismatch
Diff head:
>>>>>
--- XEET_DIR/sanity/xeet.expected/011_bad_output.out
+++ XEET_DIR/sanity/xeet.out/011_bad_output.out
@@ -1 +1 @@
-good output
+bad output
<<<<<
Full diff file: XEET_DIR/sanity/xeet.out/011_bad_output.out.diff

015c_post_command_fail                                       ....... [Failed ] Post run failed
Post run command = 1, no post run output


Finished iteration (#0/0)
Failed: 002_fail, 010_timeout, 011_bad_output, 015c_post_command_fail

